app/components/CommitteeCard.tsx

```diff
@@ -1,75 +1,102 @@
-            <a
-                href={props.committee.Site}
-                target="_blank"
-                rel="noopener noreferrer"
-            >
-              About the {props.committee.Name}
-            </a>
-          </strong>
-        </Body>
-      )
-      : <></>
-  const guide = props.committee.Site
-      ? (
-          <Body align="center" size="1rem" color={colors.primaryBlue} margins="0 0 0 0">
-            <br/>
-            <strong>
-              <Link href={`bg/${props.committee.Guide}`}>
-                <a
-                    target="_blank"
-                    rel="noopener noreferrer"
-                >Background Guide</a>
-              </Link>
-            </strong>
-          </Body>
-      )
-      : <></>
+  const link = props.committee.Site ? (
+    <Body
+      align="center"
+      size="1rem"
+      color={colors.primaryBlue}
+      margins="0 0 0 0"
+    >
+      <br />
+      <strong>
+        <a
+          href={props.committee.Site}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          About the {props.committee.Name}
+        </a>
+      </strong>
+    </Body>
+  ) : (
+    <></>
+  );
+  const guide = props.committee.Guide ? (
+    <Body
+      align="center"
+      size="1rem"
+      color={colors.primaryBlue}
+      margins="0 0 0 0"
+    >
+      <br />
+      <strong>
+        <Link href={`bg/${props.committee.Guide}`}>
+          <a target="_blank" rel="noopener noreferrer">
+            Background Guide
+          </a>
+        </Link>
+      </strong>
+    </Body>
+  ) : (
+    <></>
+  );
 
   return (
-      <MaterialCard>
-        <SealImage src={`/committees/${props.committee.Img}`}/>
-        <CardItemWrapper>
-          <Header line={0.8} mobSize="1.5rem">{props.committee.Name} {props.committee.Abbr ? ("(" + props.committee.Abbr + ")") : ""}<hr/></Header>
-          <SubTitle size="1.4rem" weight={700} line={0.5} margins="0 0 0 0" mobSize="1.2rem">Anticipated Topics:</SubTitle>
-          <br/>
-          <ol>
-            <Body size="1.1rem" line={1.5}>
-              {topics}
-            </Body>
-          </ol>
-          {guide}
-          {link}
-        </CardItemWrapper>
-      </MaterialCard>
-  )
+    <MaterialCard background={colors.ivory}>
+      <SealImage src={`/committees/${props.committee.Img}`} />
+      <CardItemWrapper>
+        <Header line={0.8} mobSize="1.5rem">
+          {props.committee.Name}{" "}
+          {props.committee.Abbr ? "(" + props.committee.Abbr + ")" : ""}
+          <hr />
+        </Header>
+        <SubTitle
+          size="1.4rem"
+          weight={700}
+          line={0.5}
+          margins="0 0 0 0"
+          mobSize="1.2rem"
+        >
+          Anticipated Topics:
+        </SubTitle>
+        <br />
+        <ol>
+          <Body size="1.1rem" line={1.5}>
+            {topics}
+          </Body>
+        </ol>
+        {guide}
+        {link}
+      </CardItemWrapper>
+    </MaterialCard>
```

Comment:
I like how readable this is, very nice very nice

Reply:
:)))